install/mysql/glpi-empty.sql

```diff
@@ -1237,6 +1237,7 @@ CREATE TABLE `glpi_impactitems` (
 	KEY `source` (`itemtype`, `items_id`),
 	KEY `parent_id` (`parent_id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
+INSERT INTO `glpi_configs` VALUES ('211','core','lock_floating_message','1');
```

Comment:
This has been moved in `install/empty_data.php` for glpi 9.5

Reply:
oups!

Reply:
I'm against adding a new preference.

This is a display detail, not something like a feature that has drastically changed.